mysite/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,22 @@
+{% extends "ct/portal.html" %}
```

Comment:
@alexbojko our new templates should extend something new but not portal template.